packages/flutter/test/widgets/editable_text_shortcuts_test.dart

```diff
@@ -2201,4 +2208,106 @@ void main() {
     }, variant: appleOnly);
 
   }, skip: !kIsWeb);// [intended] specific tests target web.
+
+  group('Web does accept', () {
+    group('desktop platforms', () {
+      const SingleActivator selectUp =
+          SingleActivator(LogicalKeyboardKey.arrowUp, shift: true);
+
+      const SingleActivator selectDown =
+          SingleActivator(LogicalKeyboardKey.arrowDown, shift: true);
+
+      testWidgets('select up', (WidgetTester tester) async {
+        controller.text = testVerticalText;
+        controller.selection = const TextSelection.collapsed(
+          offset: 5,
+        );
+
+        await tester.pumpWidget(buildEditableText());
+        await sendKeyCombination(tester, selectUp);
+        await tester.pump();
+
+        expect(controller.text, testVerticalText);
+        expect(
+          controller.selection,
+          const TextSelection(
+              baseOffset: 5,
+              extentOffset: 3), // selection extends upwards from 5
+          reason: selectUp.toString(),
+        );
+      }, variant: TargetPlatformVariant.desktop());
```

Comment:
I believe it's possible to set the `variant` on the group so you don't have to do it for each test.

Reply:
Apparently `variant` is only a param on `testWidgets` helper :(

It looks like you can only skip `group` tests